utils/admin.ts

```diff
@@ -1,6 +1,7 @@
 import type { JSONSchema7 } from "../deps.ts";
 
 export const adminDomain = `https://deco.cx`;
+export const adminAlternateDomain = `https://admin.deco.cx`;
```

Comment:
```suggestion
const adminDomains = [adminDomain, "https://admin.deco.cx"];
```

Reply:
edit line 10 to be 
```ts
  if (adminDomains.some((_adminDomain) => url.startsWith(_adminDomain))) {

```

Reply:
We use `adminDomain` to create links as well. I think it's ok